odml/property.py

```diff
@@ -373,45 +373,73 @@ def clone(self):
 
         return obj
 
-    def merge(self, other, strict=True):
+    def merge_check(self, source, strict):
```

Comment:
Good point. Did we agree on non-strict by default? I can't remember what we said, but now that I think about it, perhaps strict by default is better since it's the least destructive.

Reply:
Since this is just a check and not a method that does any actual change I don't think a default is required or necessary at this point.